kong/global.lua

```diff
@@ -167,16 +167,43 @@ end
 function _GLOBAL.init_worker_events()
   -- Note: worker_events will not work correctly if required at the top of the file.
   --       It must be required right here, inside the init function
-  local worker_events = require "resty.worker.events"
+  local worker_events
+  local opts
 
-  local ok, err = worker_events.configure {
-    shm = "kong_process_events", -- defined by "lua_shared_dict"
-    timeout = 5,            -- life time of event data in shm
-    interval = 1,           -- poll interval (seconds)
+  local singletons = require "kong.singletons"
```

Comment:
Instead of using `singletons`, can we in any way use the global `kong`, or do we have it ready at this point (e.g. `kong.worker_events` or `kong.events`?

Reply:
I checked the kong's code and have no idea about how to get the conf var, so I have to use `kong.singletons`, Do you know a better solution for it?